src/applications/mhv/medical-records/components/LabsAndTests/ChemHemDetails.jsx

```diff
@@ -209,7 +221,13 @@ const ChemHemDetails = props => {
               Compose a message.
             </a>
           </p>
-        </va-additional-info>
+          <br />
+          <p>
+            <strong>Note:</strong> if you have questions about more than 1 test
+            oredered by the same care team, send 1 message with all of your
+            questions.
```

Comment:
1. In other "Note:" sections, "if" is capitalized. Should it be the same here?
2. "oredered" is misspelled.